dev/cognitiveservices/data-plane/Language/common.json

```diff
@@ -345,9 +345,27 @@
     "RequestStatistics": {
       "type": "object",
       "required": [
+        "documentsCount",
+        "validDocumentsCount",
+        "erroneousDocumentsCount",
         "transactionsCount"
       ],
       "properties": {
+        "documentsCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of documents submitted in the request."
+        },
+        "validDocumentsCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of valid documents. This excludes empty, over-size limit or non-supported languages documents."
+        },
+        "erroneousDocumentsCount": {
+          "type": "integer",
+          "format": "int32",
```

Comment:
why are these int32 but transactionsCount is int64?

Reply:
Keeping it consistent with previously used object: https://msazure.visualstudio.com/Cognitive%20Services/_git/API-TextAnalytics/pullrequest/6387049?_a=files .

Reply:
More specifically, Service ends up using long for transactions count and int for other 3 counts.